cmd/security-secretstore-setup/Dockerfile

```diff
@@ -55,4 +55,8 @@ COPY --from=builder /edgex-go/cmd/security-secretstore-setup/entrypoint.sh /usr/
 RUN chmod +x /usr/local/bin/entrypoint.sh \
     && ln -s /usr/local/bin/entrypoint.sh /
 
+# Create token directory and assign appropriate permissions
+RUN mkdir -p /vault/config/assets && \
+    chown -Rh 100:1000 /vault/
```

Comment:
Add to comment why `100:1000`

Reply:
I didn't create that - that was added before I touched it - I just moved it around...

Reply:
it is 100 because dockerd volume mount is 100. it is 1000 because that is the first grounp in the container :-)